frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -255,24 +251,23 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
             <InfoIcon />
           </InfoButton>
         </TooltipWrapper>
-        {open &&
-          popUpContext.propertyInfo &&
-          canViewProperty &&
-          popUpContext.propertyTypeID === PropertyTypes.PARCEL && (
-            <TooltipWrapper toolTipId="associated-buildings-id" toolTip="Associated Buildings">
-              <BuildingsButton
-                id="slideOutBuildings"
-                variant="outline-secondary"
-                className={clsx({ open })}
-                onClick={() => {
-                  setAsscBuildingsOpen(true);
-                  setGeneralInfoOpen(false);
-                }}
-              >
-                <BuildingSvg className="svg" />
-              </BuildingsButton>
-            </TooltipWrapper>
-          )}
+        {open && popUpContext.propertyInfo && canViewProperty && (
+          <TooltipWrapper
+            toolTipId="associated-items-id"
+            toolTip={isBuilding ? 'Associated Land' : 'Associated Buildings'}
+          >
+            <TabButton
+              id="slideOutTab"
+              variant="outline-secondary"
+              className={clsx({ open })}
+              onClick={() => {
+                setGeneralInfoOpen(false);
+              }}
+            >
+              {isBuilding ? <LandSvg className="svg" /> : <BuildingSvg className="svg" />}
```

Comment:
this hurt my brain a bit, but looks correct given you are displaying associations here.

Reply:
yeah it is a bit weird